packages/sign-client/src/controllers/engine.ts

```diff
@@ -396,7 +396,7 @@ export class Engine extends IEngine {
     // This allows the client to use the updated session like emitting events
     // without waiting for the peer to acknowledge
     await this.client.session.update(topic, { namespaces });
-    this.sendRequest({
+    await this.sendRequest({
```

Comment:
This updates the behavior of `update` to not be as "Fire and forget" - Are there any implications of this we need to consider?

Or is this fine because of the reasoning of
"If it's fired by the queue, no big deal since the queue itself is not awaited. and if someone disabled the queue they'd want to wait"

Just want to be on the same page

Reply:
this resolves flaky tests but also make sending request(update) consistent and allows using the updated session immediately